integration-tests/src/test/resources/features/slackaction.feature

```diff
@@ -0,0 +1,42 @@
+Feature: Slack Action tests
```

Comment:
I would rename the file as `action-slack.feature` so we would have all action feature files together :)
@sutaakar wdyt ?

Reply:
Sounds good.

Reply:
Yes sounds good